### PR TITLE
Allow heif image to preview in document page

### PR DESCRIPTION
**Pull request by aburathoki**
Tested the initial changes in staging. The user can upload heif files. 
The uploaded document and its detail appear on the resident page under pending review with correct information. 
However, I realised after testing that I need to make a minor change to allow the heic file type to display in Image Preview. The current behaviour is user clicks on the document to preview, and the document is downloaded.
So, this PR includes a minor change to allow conversion of heif images to web browser compatible format
 and to display on the resident's document page.

### CODE REVIEW

File: src/components/ImagePreview.tsx [hunk 1]
```diff
@@ -40,7 +40,7 @@ const ImagePreview = (props: Props): JSX.Element | null => {
       <figure className={styles.preview}>
         {document.extension === 'jpeg' || document.extension === 'png' ? (
           <img src={downloadUrl} alt={documentSubmission.documentType.title} />
-        ) : document.extension === 'heic' ? (
+        ) : document.extension === 'heic' || document.extension === 'heif' ? (
```
Comment (bogdan-zaharia) [inline src/components/ImagePreview.tsx:43]: We're starting to see a pattern here for files with extensions that need to be converted. Might be worth considering adding them in an array and then check if the current file extension was found in that array.

Comment (bogdan-zaharia) [reply src/components/ImagePreview.tsx:43]: I know it's not part of this PR but also noticed that the `heicImage` variable name might be misleading as the `src` being passed to the `` in line 50 will sometimes have a `heif` extension.

**Review by bogdan-zaharia — COMMENTED** [review]
Left a couple comments. Happy to approve if this is not something we want to change as part of this PR.

File: src/components/ImagePreview.tsx (hunk 1 above)
Comment (aburathoki) [reply src/components/ImagePreview.tsx]: Refactored the code to allow document extension to be checked against an array of extensions that requires conversion. 
This also includes an additional change to check if the document with a certain type of extension does not require conversion.

File: src/components/ImagePreview.tsx [hunk 2]
```diff
@@ -5,50 +5,55 @@ import LoadingBox from '@govuk-react/loading-box';
...
           <LoadingBox
             loading={loading}
             title={documentSubmission.documentType.title}
           >
             <img
-              data-cy="heic-image"
-              src={heicImage}
+              src={conversionImage}
               alt={documentSubmission.documentType.title}
+              data-testid="heic-image"
```
Comment (bogdan-zaharia) [inline src/components/ImagePreview.tsx:56]: This should probably be `conversion-image` instead of `heic-image`

**Review by bogdan-zaharia — COMMENTED** [review]
Looks good, left a minor comment

File: src/components/ImagePreview.tsx (hunk 2 above)
Comment (aburathoki) [reply src/components/ImagePreview.tsx]: datatest-id value changed to "conversion-image", fixed test to reflect that.

**Review by bogdan-zaharia — APPROVED** [review]
LGTM

**Pull request merged**